integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/steps/IngressSteps.java

```diff
@@ -39,33 +61,23 @@ public void testIngressEndpoint(String cloudEvent) {
                 new Header("ce-subject", "mySubject"));
 
         String token = context.getManagerToken();
-
         IngressResource.postJsonEventResponse(token, endpoint, cloudEventStream, headers)
                 .then()
                 .statusCode(200);
 
-        cloudEventStream = new ByteArrayInputStream(cloudEvent.getBytes(StandardCharsets.UTF_8));
-        IngressResource
-                .postJsonEventResponse(token, endpoint + "/ingress/not-the-bridge-name/",
-                        cloudEventStream, headers)
-                .then()
-                .statusCode(404);
+        // TODO 
+        // cloudEventStream = new ByteArrayInputStream(cloudEvent.getBytes(StandardCharsets.UTF_8));
+        // IngressResource
+        //         .postJsonEventResponse(token, endpoint + "/ingress/not-the-bridge-name/",
+        //                 cloudEventStream, headers)
+        //         .then()
+        //         .statusCode(404);
 
-        // Plain endpoint for non cloud events payloads
-        IngressResource
-                .postPlainEventResponse(token, endpoint, "{\"data\": \"test\"}", headers)
-                .then()
-                .statusCode(200);
-    }
-
-    @And("^the Ingress is Undeployed within (\\d+) (?:minute|minutes)$")
-    public void ingressUndeployedWithinMinutes(int timeoutMinutes) {
-        String endpoint = BridgeUtils.getOrRetrieveBridgeEndpoint(context);
-
-        Awaitility.await().atMost(Duration.ofMinutes(timeoutMinutes)).pollInterval(Duration.ofSeconds(5))
-                .untilAsserted(() -> IngressResource
-                        .optionsJsonEmptyEventResponse(context.getManagerToken(), endpoint)
-                        .then()
-                        .statusCode(Matchers.anyOf(Matchers.is(404), Matchers.is(503))));
+        // // Plain endpoint for non cloud events payloads
+        // // Plain endpoint for non cloud events payloads
+        // IngressResource
+        // .postPlainEventResponse(token, endpoint, "{\"data\": \"test\"}", headers)
```

Comment:
We should test also plain endpoint, though that can be addressed as a separate JIRA.

Reply:
as per discussion will revert that and create a new issue

Reply:
revert done. Will be updated later as part of https://issues.redhat.com/browse/MGDOBR-361